axelor-supplychain/src/main/java/com/axelor/apps/supplychain/web/SupplychainSaleConfigController.java

```diff
@@ -0,0 +1,20 @@
+package com.axelor.apps.supplychain.web;
+
+import com.axelor.apps.sale.db.SaleConfig;
+import com.axelor.apps.supplychain.service.SupplychainSaleConfigServiceImpl;
+import com.axelor.exception.service.TraceBackService;
+import com.axelor.inject.Beans;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+
+public class SupplychainSaleConfigController {
+  public void updateCustomerCredit(ActionRequest request, ActionResponse response) {
+    try {
+      SaleConfig saleConfig = request.getContext().asType(SaleConfig.class);
+      Beans.get(SupplychainSaleConfigServiceImpl.class).updateCustomerCredit(saleConfig);
```

Comment:
Please, inject the interface (SupplychainSaleConfigService) instead of an implementation